lib/browser/mqtt.ts

```diff
@@ -203,6 +204,11 @@ export interface MqttConnectionConfig {
 }
 
 /**
+ * @deprecated We strongly recommend using {@link Mqtt5Client}
+ * There are no current plans to fully deprecate the MQTT 3.1.1 client but it is highly recommended customers
```

Comment:
I feel like we can shorten this along the lines of

"Use {@link Mqtt5Client} to access a more robust feature set, clearer error handling, and lifetime
 * management. More details can be found at [link]". Any other further details should be in the link and not on the repeated doc comments.